ci/workflows/main-cron.yml

```diff
@@ -527,6 +528,29 @@ steps:
     timeout_in_minutes: 20
     retry: *auto-retry
 
+  # TODO(Kexiang): Enable this test after we have a GCS_SOURCE_TEST_CONF.
```

Comment:
@tabVersion Can we set a `GCS_SOURCE_TEST_CONF` to provide credential(or `GOOGLE_APPLICATION_CREDENTIALS`) and bucket name to enable GCS mron test?

Reply:
I am not sure about the details, cc @wcy-fdu

Reply:
I'm working on this, but not easy due to some network issue.
As we already have s3_v2 e2e test and local fs source test, I think gcs test is not urgent beacuse they just share the same implemetation.

Reply:
OKK, maybe we first leave it as comments and `TODO` in this PR, since it's required in the version release. @wcy-fdu Should we also add doc for this?

Reply:
update: we have reminded our doc team to add doc for gcs source.